### PR TITLE
chore(backend): allow BE to reload when changed in dev

### PR DESCRIPTION
**Pull request by Peedee2002**
- allow BE to reload when saved 
- updated readme
- removed useless stuff from requirements.txt

HEADLINE: You will be able to run the BE in docker and not compromise!

### CODE REVIEW

File: backend/production.dockerfile [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy entire backend into the container
+COPY . .
+
+# Expose port 8000 to the outside world
+EXPOSE 8000
+
+# Run the server
+ENTRYPOINT nodemon --exec python3 -u runserver.py
```
Comment (lhjt) [inline backend/production.dockerfile]: Nodemon should not be used in production

File: frontend/README.md [hunk 2]
```diff
@@ -46,8 +46,15 @@ You will now have the backend API available on `localhost:8000/docs` and the fro
 Note that changes to the frontend React code will be visible live while developing due to the bind mount in the docker-compose file (see line 37). It is not necessary to rebuild the images to view the frontend changes.
 
 To see what containers are running, use `docker ps`. To see all containers, including stopped containers, add option `-a`. You can remove a particular container using `docker rm <container>`. Another handy command is `docker logs <containerName>` to view a container's output.
+#### Running against live backend
 
+For FE development, it is probably easier to hit the live instance of circlesapi and run the FE locally. You can do this by doing:
+```bash
+npm install # install dependancies
```
Comment (lhjt) [inline frontend/README.md]:
```suggestion
npm install # install dependencies
```

**Review by lhjt — CHANGES_REQUESTED** [review]

File: backend/production.dockerfile (hunk 1 above)
Comment (Peedee2002) [reply backend/production.dockerfile]: Oh oops I shouldn't have changed that my bad

**Review by lhjt — CHANGES_REQUESTED** [review]
Renaming the backend Dockerfile will cause the [image build pipeline](https://github.com/csesoc/Circles/blob/f43afd077f7e2a127e36150b2f7ad2ba0f701368/.github/workflows/docker.yml#L23) to fail; please update this as well

**Review by lhjt — APPROVED** [review]

**Review by sachk — APPROVED** [review]

File: frontend/README.md [hunk 3]
```diff
@@ -46,8 +46,15 @@ You will now have the backend API available on `localhost:8000/docs` and the fro
 Note that changes to the frontend React code will be visible live while developing due to the bind mount in the docker-compose file (see line 37). It is not necessary to rebuild the images to view the frontend changes.
 
 To see what containers are running, use `docker ps`. To see all containers, including stopped containers, add option `-a`. You can remove a particular container using `docker rm <container>`. Another handy command is `docker logs <containerName>` to view a container's output.
+#### Running against live backend
```
Comment (gtangelo) [inline frontend/README.md:49]: will probs clean up the frontend readme at some point but thanks for the update!

**Review by gtangelo — APPROVED** [review]

**Pull request merged**